coreos-koji-tagger/coreos_koji_tagger.py

```diff
@@ -373,27 +429,29 @@ def runcmd(cmd: list, **kwargs: int) -> subprocess.CompletedProcess:
         raise
     return cp # subprocess.CompletedProcess
 
-def get_rich_info_for_rpm_string(string: str, arch: bool) -> hawkey.NEVRA:
-    # arch: (bool) whether arch is included in the string
-    if arch:
+def get_NVRA_from_NEVRA(string: str) -> str:
```

Comment:
Everything in this function is double spaced now

Reply:
fixed in next upload